### DISCUSSION TITLE
[mini.pick] grep_live picker

### DISCUSSION THREAD

**Discussion by skoch13** [nvim-mini/mini.nvim#617] [Q&A] [answered]
Is it possible to provide `rg` with arguments on demand as https://github.com/nvim-telescope/telescope-live-grep-args.nvim allows?

**Comment by echasnovski** [comment, marked as answer]
All CLI tools are called only with basic arguments. It is suggested to [use their respective ways of configuring command](https://github.com/echasnovski/mini.nvim/blob/333d2d1090c80ac936b960469a6e93982cbaeb21/doc/mini-pick.txt#L79-L84). For `rg` it is done via creating a file and setting `RIPGREP_CONFIG_PATH` environmane variable to point at it.

If you want to have several versions of `grep_live` which use different sets of options, take a look at [this comment](https://github.com/echasnovski/mini.nvim/issues/513#issuecomment-1807907150).

**Reply by skoch13** [reply]
I read it, thanks. This approach might work when say 'filetype' one's filtering is constant, while my use case requires changing it on the fly.

**Reply by echasnovski** [reply]
No, sorry. Changing `rg` arguments on the fly is not supported.
